tests/unit/assessment/test_pipelines.py

```diff
@@ -49,9 +117,32 @@ def test_pipeline_assessment_without_config(mocker):
             state=PipelineState.IDLE,
         )
     ]
-    ws = Mock()
+    ws = create_autospec(WorkspaceClient)
     config_dict = {}
+    pipeline_cluster = [
```

Comment:
refactor to start using `workspace_client_mock`:

https://github.com/databrickslabs/ucx/blame/main/tests/unit/assessment/test_clusters.py#L22-L58

Reply:
replaced it with workspace_client_mock